commands/debug.js

```diff
@@ -62,43 +62,65 @@ if(args[0].toLowerCase()==="quest"){
   message.channel.send("This creature can't do quests.");
   return;
 }
+
+var messagePing = message.mentions.members.first();
+var targetID = messagePing ? message.guild.id.concat(messagePing.id) : userid;
```

Comment:
does this work properly? I'm used to putting conditionals in parenthesis, it'd be cool if they weren't actually required.

Reply:
I usually do, too, but the ternary operator here is the only thing on the line, and each part of the ternary operator consists of only one term.
I'll say that I tested it and it certainly seems to work just fine.